plugins/out_forward/forward.c

```diff
@@ -49,6 +54,18 @@ void _secure_forward_tls_error(struct flb_forward *ctx,
     flb_plg_error(ctx->ins, "flb_io_tls.c:%i %s", line, err_buf);
 }
 
+static int io_net_write(struct flb_upstream_conn *conn, int unused_fd,
+                        const void* data, size_t len, size_t *out_len)
+{
+    return flb_io_net_write(conn, data, len, out_len);
+}
+
+static int io_net_read(struct flb_upstream_conn *conn, int unused_fd,
+                       void* buf, size_t len)
+{
+    return flb_io_net_read(conn, buf, len);
+}
+
```

Comment:
why create local wrappers for these?

Reply:
It is abstraction layer to hide network/socket API.

I think it exposes args too much since we need to set unused args (e.g. `flb_upstream_conn` for unix socket)
It may need to define a new struct to hide args and it can be exposed utility function.